src/com/xilinx/rapidwright/ipi/BlockStitcher.java

```diff
@@ -534,7 +535,7 @@ public static void main(String[] args) {
             }
             //System.out.println(modInstName + " " + implementationIndex);
             EDIFNetlist tmp = stitched.getNetlist();
-            stitched.setNetlist(null);
+            stitched.setNetlist(EDIFTools.createNewNetlist("dummy"));
             ModuleInst mi = stitched.createModuleInst(modInstName, modImpls.get(implementationIndex));
             stitched.setNetlist(tmp);
```

Comment:
Are you sure we don't leak any references here? It seems that `createModuleInst` will modify (and reference) the "dummy" netlist; the design then switches back out to "tmp" immediately after creation.

Reply:
This is a work-around for this specific use case in order to not make changes to `Design.createModuleInst()`.  SInce this is a rare use case, we will let it go for now.